addons/account/models/account_move.py

```diff
@@ -3378,7 +3399,11 @@ def _get_refund_tax_audit_condition(self, aml):
         whether or not it comes from a refund operation.
         This is overridden by pos in order to treat returns properly.
         """
-        return aml.move_id.type in ('in_refund', 'out_refund')
+        return (
+            aml.move_id.type in ('in_refund', 'out_refund') or
+            (aml.move_id.type == 'entry' and aml.move_id.journal_id.type == 'sale' and aml.debit > 0) or
+            (aml.move_id.type == 'entry' and aml.move_id.journal_id.type == 'purchase' and aml.credit > 0)
```

Comment:
I don't think this is right (and from my test: making a misc entry equivalent to an out refund gives the wrong tax audit, even though tags are properly inverted). We are already inverting the tags when needed, so I would say instead of doing this, we should do the same as in the tax report: here https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L3360 . We should also check things from POS there, so you'll probably need a hook doing the same as on your enterprise branch, overriding it in point_of_sale module.